package.json

```diff
@@ -49,7 +49,8 @@
   "dependencies": {
     "axios": "^0.27.2",
     "dotenv": "^16.0.1",
-    "ethers": "^5.6.9"
+    "ethers": "^5.6.9",
+    "path": "^0.12.7"
```

Comment:
path is a core lib. This one is "an exact copy of the NodeJS ’path’ module published to the NPM registry.". Is it necessary to have it as a dependency?

Reply:
i have to use this one instead:
https://www.npmjs.com/package/@faker-js/faker

Reply:
yes you're right it's a core dependency